rs/registry/canister/src/registry.rs

```diff
@@ -327,39 +326,22 @@ impl Registry {
         self.check_global_state_invariants(mutations.as_slice());
     }
 
-    /// Serializes the registry contents using the specified version of stable
-    /// representation.
-    fn serializable_form_at(&self, repr_version: ReprVersion) -> RegistryStableStorage {
```

Comment:
This function has been deleted.

If you only delete this function, you will see that (other than tests), this is only called by `serializable_form`.